pkgs/top-level/all-packages.nix

```diff
@@ -22630,7 +22630,7 @@ with pkgs;
     });
 
   libsForQt5 = (recurseIntoAttrs (import ./qt5-packages.nix {
-    inherit lib __splicedPackages makeScopeWithSplicing' generateSplicesForMkScope pkgsHostTarget;
+    inherit lib config __splicedPackages makeScopeWithSplicing' generateSplicesForMkScope pkgsHostTarget;
```

Comment:
Unnecessary?

Reply:
It is required to access config.allowAliases in applications/kde